src/components/Meeting.tsx

```diff
@@ -116,16 +113,6 @@ export default function Meeting({
     document.title = meeting.name;
   }
 
-  // feedback URL link
-  if (!meeting.feedback_url && feedback_emails.length) {
-    meeting.feedback_url = formatFeedbackEmail(
-      settings.feedback_emails,
-      meeting,
-      settings,
-      strings
-    );
-  }
-
```

Comment:
to keep this page a little simpler we're forming the `feedback_url` in `load-meeting-data`